torch/nn/quantized/functional.py

```diff
@@ -452,6 +452,25 @@ def hardswish(input, scale, zero_point):
         raise ValueError("Input to 'quantized.hardswish' must be quantized!")
     return torch._ops.ops.quantized.hardswish(input, scale, zero_point)
 
+def threshold(input, threshold, value):
+    # type: (Tensor, float, float) -> Tensor
+    r"""Applies the quantized version of the threshold function element-wise:
+
+    .. math::
+        \text{threshold}(x) = x \text{ if } x > \text{threshold else } \text{ value}
+
+    Args:
+        input: quantized input
+        threshold: the threshold value used to determine if :math:`x` should be
+        replaced by value
+        value: the value that is used in place of the original entries in x
+    """
+    if not input.is_quantized:
+        raise ValueError("Input to 'quantized.threshold' must be quantized!")
+    if (threshold is not None) != (value is not None):
+        raise ValueError("Either both or none of (threshold, value) must be specified!")
```

Comment:
These arguments are non-optional, so both have to be specified

Reply:
Sounds good! To be more specific, I've split the conditions instead into separate checks for `threshold` and `value` to make sure the inputs to both arguments must be specified.